src/RamCondition.h

```diff
@@ -492,4 +492,20 @@ inline std::unique_ptr<RamCondition> toCondition(const std::vector<const RamCond
     return result;
 }
 
+/**
+ * @brief Convert list of conditions to a conjunction
+ * @param A list of RAM conditions
+ * @param A RAM condition
+ *
+ * Convert a list {C1, C2, ..., Cn} to a condition of
+ * the format C1 /\ C2 /\ ... /\ Cn.
+ */
+inline std::unique_ptr<RamCondition> toCondition(const std::vector<std::unique_ptr<RamCondition>>& list) {
```

Comment:
Can this function be replaced with `toCondition(toConstPtrVector(conds))` in RamTransforms.cpp (as done in Synthesiser.cpp)?

It might be better to redesign `toConjuctionList`/`toCondition`, but that would be a separate PR.

Reply:
Yes, indeed. We should raise this as an issue and prepare a later PR to sort this inconsistency out.